css/layout.css

```diff
@@ -18,6 +18,12 @@
   width: 100%;
 }
 
+@media only screen and (min-width: 576px) {
+  .column-half {
+    flex-basis: 50%;
+  }
+}
+
```

Comment:
This shouldn't be needed. I think you added this because your zoom level was off

Reply:
LOL makes sense now.